A3A/addons/core/functions/Ammunition/fn_equipmentIsValidForCurrentModset.sqf

```diff
@@ -8,7 +8,7 @@ if (_configClass in A3U_forbiddenItems && {getNumber (configFile >> "A3U" >> "fo
 
 if ("specialGM" in A3A_factionEquipFlags) exitWith {
 	private _cfgName = configName _configClass;
-	if (_cfgName in ["gm_g36a1_blk", "gm_g36a1_des", "gm_g36e_blk", "gm_p2a1_launcher_blk", "gm_itemAttachment_suppressor_base"]) exitWith {false};
+	if (_cfgName in ["gm_g36a1_blk", "gm_g36a1_des", "gm_g36e_blk", "gm_p2a1_launcher_blk", "gm_itemAttachment_suppressor_base"]) exitWith {false}; ///what it does? why?
```

Comment:
Honestly no idea why, my best guess is that maybe they were spawning in crates when they shouldn't

Reply:
I'll check it, maybe I'll remove it all together

Reply:
```suggestion
	if (_cfgName in ["gm_itemAttachment_suppressor_base"]) exitWith {false};
```
Try this, there must be a reason it's there